src/test/java/cruise/umple/sample/downloader/ConsoleMainTest.java

```diff
@@ -17,19 +14,16 @@
 import com.google.inject.Inject;
 import com.google.inject.Provider;
 
-import cruise.umple.sample.downloader.ConsoleMain.ImportRuntimeData;
+import cruise.umple.sample.downloader.consistent.ImportRepository;
+import cruise.umple.sample.downloader.consistent.ImportRepositorySet;
 import cruise.umple.sample.downloader.util.MockDocumentFactoryModule;
-import cruise.umple.sample.downloader.util.Pair;
 
 /**
  * Created by kevin on 15-02-23.
  */
-@Guice(modules={DownloaderModule.class, MockDocumentFactoryModule.class})
+@Guice(modules={MockDocumentFactoryModule.class})
```

Comment:
It is our understanding that Umple doesn't support annotations, but if we read the manual we might find that indeed it does (hence one of the options to better read the manual).

As for a path forward, if we look at the modeling aspect of annotations, not necessarily the program specific needs, then we can apply it as best as possible to all languages.  So Java would have direct support, and for others it might just be represented as comments.